### PR TITLE
Add support for Hisense AEH-W4A1

### PR DESCRIPTION
**Pull request by bannhead**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28641

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -124,4 +124,4 @@ RUBY VERSION
    ruby 2.6.2p47
 
 BUNDLED WITH
-   2.0.1
+   2.0.2
```
Comment (frenck) [inline Gemfile.lock]: This is unrelated to your PR, furthermore, upgrading this will cause the builds to fail.
Please revert/remove this change.

File: source/_integrations/hisense_aehw4a1.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Hisense AEH-W4A1"
+description: "Instructions to setup the Hisense AEH W4A1 wifi module for ACs."
+logo: hisense.png
+ha_release: 0.102
+ha_category:
+  - Climate
+ha_iot_class: Local Pull
```
Comment (frenck) [inline source/_integrations/hisense_aehw4a1.markdown]:
```suggestion
ha_iot_class: Local Poll
```

File: source/_integrations/hisense_aehw4a1.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+The Hisense AEH-W4A1 is wifi module used to give wifi connectivity to some Hisense ACs and rebranded models (Smart Cool, Beko and others).
```
Comment (frenck) [inline source/_integrations/hisense_aehw4a1.markdown]:
```suggestion
The Hisense AEH-W4A1 is a WiFi module used to give WiFi connectivity to some Hisense ACs and rebranded models (Smart Cool, Beko and others).
```

File: source/_integrations/hisense_aehw4a1.markdown [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Later AC models use other wifi modules (like AEH-W4B1 and AEH-W4E1) that are not compatible or otherwise tested  with this integration.
```
Comment (frenck) [inline source/_integrations/hisense_aehw4a1.markdown]:
```suggestion
Later AC models use other WiFi modules (like AEH-W4B1 and AEH-W4E1) that are not compatible or otherwise tested with this integration.
```

File: source/_integrations/hisense_aehw4a1.markdown [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+## Configuration
+
+To integrate one or more Hisense AEH-W4A1 controlled ACs with Home Assistant, use the Configuration/Integrations or add the following section to your `configuration.yaml` file:
+
+```yaml
+# Automatic discovery
+hisense_aehw4a1:
+```
+
+Alternately, devices that are not discoverable can be statically configured.
```
Comment (frenck) [inline source/_integrations/hisense_aehw4a1.markdown:26]: Please move addition examples in a separate section below the `{% configuration %}` block.

File: source/_integrations/hisense_aehw4a1.markdown [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+Use of static configuration disables automatic discovery.
```
Comment (frenck) [inline source/_integrations/hisense_aehw4a1.markdown]:
```suggestion
The use of static configuration disables automatic discovery.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @bannhead!

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/hisense_aehw4a1.markdown (hunk 5 above)
Comment (bannhead) [reply source/_integrations/hisense_aehw4a1.markdown:26]: Only the example configuration or all the text?

Comment (frenck) [reply source/_integrations/hisense_aehw4a1.markdown:26]: I would say the text that belongs to the example should go down with the example?

**Comment by bannhead** [comment]
The parent is ready for merging. What I have to do to finish documentation?

**Comment by klaasnicolaas** [comment]
Just waiting until we have (re)reviewed your PR 😉

**Review by frenck — CHANGES_REQUESTED** [review]
There are tons of changes made to the `Gemfile.lock`, which is not related to this PR. Please revert this.

**Review by frenck — APPROVED** [review]
Thanks for the PR @bannhead! 👍

**Comment by scaramatto** [comment]
Hi, Please it's possible to add compatibility with AEH-W4B1? Thanks
